src/backer/backering.py

```diff
@@ -473,7 +474,14 @@ def on_post(self, req, rep):
             if self.ledger:
                 try:
                     event = eventing.loadEvent(self.hab.db, pre, serder.saidb)
-                    self.ledger.publishEvent(event)
+
+                    # KEL will push to on-chain icpMessage-controllerSig-witnessSig
+                    # parse witnessSig
+                    data = crt.decode()
+                    find_witness_sig = re.search("\}\-([A-z0-9\_\-]+)", data)
+                    if find_witness_sig:
+                        witness_sig = find_witness_sig.group(1)
```

Comment:
I would prefer to use the proper keripy functions to add the wigers properly rather than using our own regex which might break.

I mentioned on slack that we could try to send `msg` and `bytes(rct)` separately rather than concatenated them first. And then try to concatenate using the proper keripy code later as an optimisation - can we try that first?

Reply:
hi @iFergal 

hmm, this should be the correct way for parsing for both msg, and byte(rct)

![image](https://github.com/cardano-foundation/cardano-backer/assets/151806989/f3a90362-58f3-47fa-a3a3-7b3248c4c32e)

ref here
https://github.com/cardano-foundation/keripy/blob/7fd7f30661820cf8f514e201fdaceb62fd50bdad/src/keri/core/parsing.py#L631

But, we may need to add another version for return the `wigers`

Reply:
ah, got your point, will update the logic then verify at flushQueue
pls skip my msg above